build/linux/dist/install.sh

```diff
@@ -205,6 +205,14 @@ simple_uninstall_f() {
 # Update desktop file and mime databases (if possible)
 updatedbs_f() {
 
+  if [ if "${HOME}/root/.config/mimeapps.list"]
+    then
```

Comment:
Please be consistent with the formatting convention used in the rest of the file (`if [ ... ]; then`)